proto/aserto/directory/reader/v3/reader.proto

```diff
@@ -226,28 +224,39 @@ message GetObjectsResponse {
     aserto.directory.common.v3.PaginationResponse page          = 9     [(google.api.field_behavior) = OUTPUT_ONLY];
 }
 
-message GetRelationRequest {
+message GetObjectRelationsRequest {
     // object type
     string object_type                                          = 1     [(google.api.field_behavior) = REQUIRED];
     // object identifier
     string object_id                                            = 2     [(google.api.field_behavior) = REQUIRED];
-    // relation name
-    string relation                                             = 3     [(google.api.field_behavior) = REQUIRED];
-    // subject type
-    string subject_type                                         = 4     [(google.api.field_behavior) = REQUIRED];
-    // subject identifier
-    string subject_id                                           = 5     [(google.api.field_behavior) = REQUIRED];
-    // optional subject relation name
+    // diretion of relations to get
+    RelationsTo relations_to                                    = 3     [(google.api.field_behavior) = REQUIRED];
+    // optional filter on object type
+    string neighbor_type                                        = 4     [(google.api.field_behavior) = OPTIONAL];
```

Comment:
Would relation_object_type be more descriptive? neighbor_type would introduce a completely new term we nowhere else use.

Reply:
I thought that the term `object_type` in the name might be confusing because in GetRelations `object_type` means the type on the object side of the relations (the source_type in the relations table) and here it could be either the object or subject type.

Reply:
It was unclear what the neighbor_type is a neighbor is of.

It is the object_type of the relation, hence relation_object_type, which is a concatenation naming we use, similar to subject_relation.